### DISCUSSION TITLE
Only run specific containers from docker compose

### DISCUSSION THREAD

**Discussion by Penderis** [supabase/supabase#23611] [Questions]
Hi, I could not find a resource for this question , so trying here. 
How can I only run the required features needed by supabase via docker compose?
I would like to be able to be able to not have all 12 services running for example there is no need for analytics or the vector thing and I guess other optional stuff like realtime. 

Most answers I found have to do with the config.toml, but this seems to be for CLI and that in turn is just for local development , which well docker compose does the same thing... 

In Laravel Sail I can for example choose the services I want to launch via compose , is that possible in supabase?

**Comment by bcb** [comment]
You can specify the containers to start:
```sh
docker compose up -d db auth rest kong
```

However it will still bring up other containers those containers depend on. So you may have to edit the compose file to remove dependencies.
